docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -80,6 +81,40 @@ There are specific pages about contributing to various parts of the Gatsby ecosy
 
 For the Gatsby blog, it's necessary to run your content idea by the Gatsby team before submitting it. For more information, refer to the page on [blog and website contributions](/contributing/blog-and-website-contributions/).
 
+## Update your fork with the latest Gatsby changes
+
+The Gatsby GitHub repo is very active, so it's likely you'll need to update your fork with the latest changes to be able to merge in your code. This requires adding Gatsby as an [upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork):
+
+- Set Gatsby's repo URL as a remote source. The name of the remote is arbitrary; this example uses `upstream`.
+  ```bash
+  git remote set-url upstream git@github.com:gatsbyjs/gatsby.git
+  ```
+  - _Note: this syntax [uses SSH: you can also use `https`](https://help.github.com/en/articles/which-remote-url-should-i-use) and your username/password._
+- You can also verify the remote name and URL at any time:
+  ```bash
+  git remote -v
+  ```
+- Fetch the latest changes from Gatsby:
+  ```bash
+  git fetch upstream master
+  ```
+- In the branch you want to update, merge any changes from Gatsby into your fork:
+  ```bash
+  git merge upstream master
+  ```
+  - If there are any [merge conflicts](https://help.github.com/en/articles/resolving-a-merge-conflict-on-github), you'll want to address those to get a clean merge.
+- Once your branch is in good working order, push the changes to your fork:
+  ```bash
+  git push origin head
+  ```
+
+For more information on working with upstream repos, [visit the GitHub docs](https://help.github.com/en/articles/configuring-a-remote-for-a-fork).
+
+_**Note:** as a member of the Gatsby repo, you can also clone it directly and push to feature branches._
```

Comment:
Maybe add some hint what I have to change with the commands if Ihave fetch and merge into
A feature branch?